homeassistant/components/homewizard/const.py

```diff
@@ -11,7 +11,7 @@
 
 DOMAIN = "homewizard"
 COORDINATOR = "coordinator"
-PLATFORMS = ["sensor"]
+PLATFORMS = ["sensor", "switch"]
```

Comment:
```suggestion
PLATFORMS = [Platform.SENSOR, Platform.SWITCH]
```

with `from homeassistant.const import Platform`

Reply:
d774b4c